### ISSUE TITLE
Rename http_port to http as it is in neuro-cli

### ISSUE THREAD

**Issue by atemate** [neuro-inc/neuro-flow#309]
Since we have `neuro run ... --http 8080`, it'd be nice to have the same name in yaml.
```
kind: live
jobs:
  server:
    image: python:3.7
    preset: cpu-small
    http_port: 8080
    bash: python -m http.server 8080
```
Very obscure error message:
```
$ neuro-flow run server
ERROR: while constructing a 'Base'
  in "/home/ay/github/temp/neuro-flow/.neuro/live.yaml", line 4, column 5
unexpected key 'http'
  in "/home/ay/github/temp/neuro-flow/.neuro/live.yaml", line 6, column 5
ERROR: Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fbf24810ed0>
```

**Comment by romasku** [comment]
I would suggest the opposite - how about adding the `--http-port` option to neuro-cli and (maybe) deprecating the `--http` option. @asvetlov, what do you think about it?

The error message is indeed obscure. I think we fixed the `Unclosed client session` is fixed in `v21.3.3`, are you using the latest version? The `'Base'` will be replaced with `'JobBase'`.

**Comment by romasku** [comment]
Fixed here https://github.com/neuro-inc/neuro-cli/pull/2501
